tests/pytest_tests/system_tests/test_artifacts/test_wandb_artifacts_full.py

```diff
@@ -225,11 +225,13 @@ def test_edit_after_add(wandb_init):
 
 def test_uploaded_artifacts_are_unstaged(wandb_init, tmp_path, monkeypatch):
     # Use a separate staging directory for the duration of this test.
-    monkeypatch.setenv("WANDB_DATA_DIR", str(tmp_path))
-    staging_dir = Path(wandb_sdk.internal.artifacts.get_staging_dir())
+    monkeypatch.setenv("WANDB_DATA_DIR", str(tmp_path / "staging"))
```

Comment:
Not sure why you added the subdir, can you explain?

Reply:
I think I was trying to debug why the dir_size check was >1MB, but changing it didn't help. No need for it.